src/include/distributed/insert_select_planner.h

```diff
@@ -17,13 +17,22 @@
 #include "postgres.h"
 
 #include "distributed/multi_physical_planner.h"
+#include "distributed/multi_planner.h"
 #include "nodes/execnodes.h"
 #include "nodes/parsenodes.h"
 #include "nodes/plannodes.h"
 
 
+extern bool InsertSelectIntoDistributedTable(Query *query);
 extern MultiPlan * CreateCoordinatorInsertSelectPlan(Query *originalQuery);
+extern Query * ReorderInsertSelectTargetLists(Query *originalQuery,
+											  RangeTblEntry *insertRte,
+											  RangeTblEntry *subqueryRte);
 extern void CoordinatorInsertSelectExplainScan(CustomScanState *node, List *ancestors,
```

Comment:
It should be in multi_explain.c, since it's comparable to `CitusExplainScan` and calls static explain functions that need to remain static (due to name collisions with postgres).  Strangely, `CitusExplainScan` is declared in `multi_executor.h`